test/SlowTests/Issues/EncryptedDatabaseGroup.cs

```diff
@@ -28,12 +28,14 @@ public async Task AddingNodeToEncryptedDatabaseGroupShouldThrow()
             var options = new Options
             {
                 Server = leader,
-                ReplicationFactor = 2,
+                ReplicationFactor = 1,
```

Comment:
was this intentional?

Reply:
That's for debugging, all of those fixes need to be applied to 4.2 anyway.